appveyor.yml

```diff
@@ -11,41 +11,71 @@ environment:
         # See: http://stackoverflow.com/a/13751649/163740
         CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
         GDAL_HOME: "C:\\gdal"
-        PYTHON_ARCH: "64"
+        PYTHONWARNINGS: "ignore:DEPRECATION::pip._internal.cli.base_command"
 
-    matrix:
-        # The 4-digit number in the GISInternals archives is the MSVC version used to build
-        # the libraries. It does not need to match the version of MSVC used to build Python.
-        # https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering
 
```

Comment:
I think this comment is useful. I think it was already missing in my PR, maybe I accidentally deleted it.